### DISCUSSION TITLE
"card-image.html:12:18": execute of template failed" error

### DISCUSSION THREAD

**Discussion by dobaret** [thuliteio/doks#873] [Q&A]
Hello,

My website has been running into an issue since, I believe, the last Doks update, see: [https://github.com/dobaret/dobaret.github.io/runs/7896255655?check_suite_focus=true](https://github.com/dobaret/dobaret.github.io/runs/7896255655?check_suite_focus=true)

`ERROR 2022/08/18 09:59:23 render of "term" failed: execute of template failed: template: _default/list.html:14:9: executing "main" at <partial "content/card-image.html" .>: error calling partial: "/home/runner/work/dobaret.github.io/dobaret.github.io/node_modules/@hyas/doks/layouts/partials/content/card-image.html:12:18": execute of template failed: template: partials/content/card-image.html:12:18: executing "partials/content/card-image.html" at <$image.Resize>: error calling Resize: must provide Width or Height`

`Error: Error building site: failed to render pages: render of "section" failed: execute of template failed: template: _default/list.html:14:9: executing "main" at <partial "content/card-image.html" .>: error calling partial: "/home/runner/work/dobaret.github.io/dobaret.github.io/node_modules/@hyas/doks/layouts/partials/content/card-image.html:12:18": execute of template failed: template: partials/content/card-image.html:12:18: executing "partials/content/card-image.html" at <$image.Resize>: error calling Resize: must provide Width or Height`

I can't figure this issue out, and I'd be very grateful if someone could help me.

**Comment by h-enk** [comment]
A little late — sorry about that — but It looks like you got it fixed?

**Reply by dobaret** [reply]
Hello, unfortunately not 😅. I've been approving some automatic PRs in the hope that they could fix it, but to no avail.
